shiftleft.yml

```diff
@@ -9,4 +9,4 @@ build_rules:
       - SEVERITY_MEDIUM_IMPACT
       - SEVERITY_HIGH_IMPACT
       - SEVERITY_LOW_IMPACT
-    threshold: 10
+    threshold: 13
```

Comment:
@ursachec Most people would just copy-paste sample configuration so it is better to leave this at 0.